backend/app/api/endpoints/kind/skills.py

```
@@ -879,46 +879,108 @@ def list_unified_skills(
     User/group skills with same name take precedence over public skills.
     """
-    from app.services.group_permission import get_user_groups
+    from app.services.group_permission import (
+        get_effective_role_in_group,
+        get_user_groups,
+    )
+
+    # Validate scope parameter
+    if scope not in ("personal", "group", "all"):
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
```

Ruff (0.15.5)

[error] Undefined name `status`

(F821)